ncdump/CMakeLists.txt

```diff
@@ -37,37 +37,37 @@ set_property(SOURCE dumplib.c
   PROPERTY
     SKIP_UNITY_BUILD_INCLUSION ON)
 
-ADD_EXECUTABLE(ncdump ${ncdump_FILES})
-ADD_EXECUTABLE(nccopy ${nccopy_FILES})
-ADD_EXECUTABLE(ncvalidator ${ncvalidator_FILES})
-ADD_EXECUTABLE(ncpathcvt ${ncpathcvt_FILES})
-ADD_EXECUTABLE(ncfilteravail ${ncfilteravail_FILES})
-
-IF(USE_HDF5)
-  ADD_EXECUTABLE(nc4print nc4print.c nc4printer.c)
-  ADD_EXECUTABLE(printfqn  ${printfqn_FILES})
-  ADD_EXECUTABLE(nchdf5version ${nchdf5version_FILES})
-ENDIF(USE_HDF5)
-
-IF(ENABLE_DAP)
-  ADD_EXECUTABLE(ocprint ${ocprint_FILES})
-ENDIF(ENABLE_DAP)
-
-TARGET_LINK_LIBRARIES(ncdump netcdf ${ALL_TLL_LIBS})
-TARGET_LINK_LIBRARIES(nccopy netcdf ${ALL_TLL_LIBS})
-TARGET_LINK_LIBRARIES(ncvalidator netcdf ${ALL_TLL_LIBS})
-TARGET_LINK_LIBRARIES(ncpathcvt netcdf ${ALL_TLL_LIBS})
-TARGET_LINK_LIBRARIES(ncfilteravail netcdf ${ALL_TLL_LIBS})
-
-IF(USE_HDF5)
-  TARGET_LINK_LIBRARIES(nc4print netcdf ${ALL_TLL_LIBS})
-  TARGET_LINK_LIBRARIES(printfqn netcdf ${ALL_TLL_LIBS})
-  TARGET_LINK_LIBRARIES(nchdf5version netcdf ${ALL_TLL_LIBS})
-ENDIF(USE_HDF5)
-
-IF(ENABLE_DAP)
-  TARGET_LINK_LIBRARIES(ocprint netcdf ${ALL_TLL_LIBS})
-ENDIF(ENABLE_DAP)
+add_executable(ncdump ${ncdump_FILES})
+add_executable(nccopy ${nccopy_FILES})
+add_executable(ncvalidator ${ncvalidator_FILES})
+add_executable(ncpathcvt ${ncpathcvt_FILES})
+add_executable(ncfilteravail ${ncfilteravail_FILES})
+
+if(USE_HDF5)
+  add_executable(nc4print nc4print.c nc4printer.c)
+  target_link_libraries(nc4print netcdf ${ALL_TLL_LIBS})
+
+  add_executable(printfqn  ${printfqn_FILES})
+  target_link_libraries(printfqn netcdf ${ALL_TLL_LIBS})
+
+  add_executable(nchdf5version ${nchdf5version_FILES})
+  target_link_libraries(nchdf5version netcdf ${ALL_TLL_LIBS})
+endif(USE_HDF5)
+
+if(ENABLE_DAP)
+  add_executable(ocprint ${ocprint_FILES})
+endif(ENABLE_DAP)
+
+message(STATUS "ALL_TLL_LIBS INCLUDE FOR KYLE: ${ALL_TLL_LIBS}")
```

Comment:
Was this just for debugging? Could go now?

Reply:
correct. Fixed.